### ISSUE TITLE
{{whereSnippet}} in controller.stub not working

### ISSUE THREAD

**Issue by daemswibowo** [appzcoder/crud-generator#151]
i just create new controller for PermissionController, when i run in browser it return error on line 26. this is PermissionController.php script:
```
    public function index(Request $request)
    {
        $keyword = $request->get('search');
        $perPage = 25;

        if (!empty($keyword)) {
            $permission = Permission::
                ->paginate($perPage);
        } else {
            $permission = Permission::paginate($perPage);
        }

        return view('dashboard.manajemen-pengguna.permission.index', compact('permission'));
    }
```

in Controller.stub, {{whereSnippet}} does'nt work
```
    public function index(Request $request)
    {
        $keyword = $request->get('search');
        $perPage = {{pagination}};

        if (!empty($keyword)) {
            ${{crudName}} = {{modelName}}::{{whereSnippet}}
                ->paginate($perPage);
        } else {
            ${{crudName}} = {{modelName}}::paginate($perPage);
        }

        return view('{{viewPath}}{{viewName}}.index', compact('{{crudName}}'));
    }
```

**Comment by sohelamin** [comment]
@daemswibowo for now put any field on `where` part to make it working I'll fix it on the next release
eg.
```php
$permission = Permission::where('name', 'LIKE', "%$keyword%")
                ->paginate($perPage);
```

**Comment by sohelamin** [comment]
Fixed at 9b35da4

**Comment by daemswibowo** [comment]
Thanks for this great tool @sohelamin
